CMakeLists.txt

```diff
@@ -366,7 +366,7 @@ if (${use_prov_client_core})
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_PROV_MODULE")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DUSE_PROV_MODULE")
 
-    if (${build_provisioning_service_client} AND ${use_prov_client})
+    if ((${build_provisioning_service_client} AND ${use_prov_client}) OR ${run_e2e_tests})
```

Comment:
> OR ${run_e2e_tests} [](start = 69, length = 20)

This is interesting logic here.  So if you don't have e2e test you you won't have provisioning service client.  I like the idea, but I'm not sure if others will go with it.  I just want to make sure you meant to have that.

Reply:
Yes, this is for the case of building for Edge where ${use_prov_client_core} is ON and not ${use_prov_client}.  We would only bring provisioning_service_client for the case of e2e tests